spec/features/admin_views_admin_panel_spec.rb

```diff
@@ -1,37 +1,41 @@
 require 'rails_helper'
 
-feature 'user views homepage', %Q{
+feature 'admin views admin panel', %Q{
```

Comment:
Use `%` instead of `%Q`.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.<br>Use `%Q` only for strings that contain both single quotes and double quotes, or for dynamic strings that contain double quotes.